superset/security/manager.py

```diff
@@ -14,8 +14,10 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-# pylint: disable=too-many-lines
+# pylint: disable=too-many-lines, invalid-name
```

Comment:
I don't think we should use the invalid-name rule as well... 
I use the clean code principles, so there is no sense to put ignoring on each statement

Reply:
Putting on each statement means you're aware of the rule, and you consciously decided to break it because practicality beats purity. If you put it at the top of the model the rule could be broken many times without people being aware of it, which reduces the code quality.

Reply:
+1. The desired pattern is mentioned [here](https://github.com/apache/superset/blob/master/CONTRIBUTING.md#python).

Reply:
I'll add and open new discussion on it